### PR TITLE
Add newer versions of Django and Python to CI.

### PR DESCRIPTION
**Pull request by D3X**

### CODE REVIEW

**Comment by simkimsia** [comment]
@jmcarp Please merge this as I am using py37 and django22 and I would like to use this library

Thank you

**Comment by hartwork** [comment]
@jmcarp any thoughts?

File: tox.ini [hunk 1]
```diff
@@ -25,6 +27,8 @@ deps=
     django111: django>=1.11,<2.0
     django20: django>=2.0,<2.1
     django21: django>=2.1,<2.2
+    django22: django>=2.2,<3.0
+    django30: django>=3.0,<3.1
```
Comment (aleyan) [inline tox.ini:31]: Since Django 3.1 has been released in August, should it be included?

Comment (D3X) [reply tox.ini:31]: I completely forgot about this PR (I have opened it in March). Will add 3.1 in a sec :)

Comment (simkimsia) [reply tox.ini:31]: Will this be merged now you're reviewing it @aleyan ?

Comment (aleyan) [reply tox.ini:31]: Thank you @D3X . @simkimsia Would love to merge this, but unfortunately I don't have access, thats up to @jmcarp .

**Comment by D3X** [comment]
There is a newer PR, that adds Django 3.2 to the test matrix: #40. Closing.

**Pull request closed without merging**